src/cloudant/database.py

```diff
@@ -1291,7 +1291,7 @@ def get_search_result(self, ddoc_id, index_name, **query_params):
         :param int limit: Optional number to limit the maximum count of the
             returned documents. In case of a grouped search, this parameter
             limits the number of documents per group.
-        :param query: A Lucene query in the form of ``name:value``.
+        :param q or query: A Lucene query in the form of ``name:value``.
```

Comment:
I don't think that this is valid Sphinx syntax so it may cause Sphinx to issue a warning.  If it does then I would separate the `q` param and the `query` param and then add a mention that only one of them is needed.